netbox_topology_views/views.py

```diff
@@ -238,6 +238,7 @@ def create_edge(
     edge_id: int,
     termination_a: Dict,
     termination_b: Dict,
+    straight_cables: bool,
```

Comment:
Shouldn't this be an optional parameter? I'd use the Type Hint Syntax like in the other parameters below.